packages/react/__tests__/get-label-from-stack-trace.js

```diff
@@ -0,0 +1,560 @@
+// @flow
+import {
+  getFunctionNameFromStackTraceLine,
+  getLabelFromStackTrace
+} from '../src/get-label-from-stack-trace'
+
+describe('getFunctionNameFromStackTraceLine', () => {
+  test('V8', () => {
+    expect(getFunctionNameFromStackTraceLine('Error')).toBeUndefined()
+
+    expect(
+      getFunctionNameFromStackTraceLine(
+        '    at Object.createEmotionProps (emotion-element-1fb5ab00.browser.esm.js:143)'
+      )
+    ).toBe('createEmotionProps')
+  })
+
+  test('Safari / Firefox', () => {
+    expect(
+      getFunctionNameFromStackTraceLine(
+        'createEmotionProps@http://localhost:3000/static/js/main.chunk.js:844:40'
+      )
+    ).toBe('createEmotionProps')
+  })
+})
```

Comment:
I probably think that this is a little bit redundant because it's sort of an implementation detail. It makes sense that we have to test the whole thing using the unit test approach because there is no other way to do it. However, I feel like we should always try to focus on the outermost "boundary". There are cases when units makes sense - sometimes it's just nice to isolate some tricky, convoluted, logic and test it in separation. Especially if it is consumed by many other places in the code.

This is just a nit, and a personal preference - so it's fine as is. It just kinda doesn't match the style of testing used in the rest of the codebase.